linear_algebra/gram_schmidt.cpp

```diff
@@ -0,0 +1,245 @@
+/**  
+ * @file   
+ * @brief Gram Schimdt Orthogonalisation Process (https://en.wikipedia.org/wiki/Gram%E2%80%93Schmidt_process)
+ *
+ * @details
+ * Takes the input of Linearly Independent Vectors,
+ * returns vectors orthogonal to each other.
+ * 
+ * ### Algorithm
+ * Take the first vector of given LI vectors as first vector of Orthogonal vectors.
+ * Take projection of second input vector on the first vector of Orthogonal vector 
+ * and subtract it from the 2nd LI vector.
+ * Take projection of third vector on the second vector of Othogonal vectors and subtract it from the 3rd LI vector.
+ * Keep repeating hte above process until all the vectors in the given input array are exhausted. 
+ *
+ * For Example:
+ * In R2, 
+ * Input LI Vectors={(3,1),(2,2)}
+ * then Orthogonal Vectors= {(3, 1),(-0.4, 1.2)}
+ *
+ *  Have defined maximum dimension of vectors to be 10 and number of vectors
+ *  taken is 20.
+ *  Please do not give linearly dependent vectors
+ *
+ *
+ * @author [Akanksha Gupta](https://github.com/Akanksha-Gupta920)
+ */
+
+#include <iostream>
+#include <cassert>
+#include <cmath>
+#include <array>
```

Comment:
Add a brief description of what `cmath` is added (see the other examples).
```suggestion
#include <iostream>  /// for io operations
#include <cassert>  /// for assert
#include <cmath>   ///
#include <array>  /// for std::array
```